elcid/patient_lists.py

```diff
@@ -153,25 +157,53 @@ class Bacteraemia(RfhPatientList):
     order = -10
 
 
-class PCP(RfhPatientList):
+class PCP(RfhPatientList, TaggedPatientList):
     display_name = 'PCP'
     direct_add = True
     tag = "pcp"
     template_name = 'episode_list.html'
     schema = []
 
 
-class R1(RfhPatientList):
+class R1(RfhPatientList, TaggedPatientList):
     display_name = 'R1'
     direct_add = True
     tag = "r1"
     template_name = 'episode_list.html'
     schema = []
 
 
-class LiverTransplantation(RfhPatientList):
+class LiverTransplantation(RfhPatientList, TaggedPatientList):
     display_name = 'Liver Transplantation'
     direct_add = True
     tag = "liver_transplantation"
     template_name = 'episode_list.html'
     schema = []
+
+
+class ChronicAntifungal(RfhPatientList, PatientList):
+    display_name = "Chronic Antifungal"
+    template_name = 'episode_list.html'
+    schema = []
+    is_chronic = True
+
+    @property
+    def queryset(self):
+        active_from_date = datetime.date.today() - datetime.timedelta(3)
```

Comment:
I'd be tempted to avoid the second query and re-sorting e.g. 

```
 def get_queryset(self, user):
     active_from_date = datetime.date.today() - datetime.timedelta(3)
     episodes = omodels.Episode.objects.filter(
         patient__chronicantifungal__updated_dt__gte=active_from_date
     ).filter(
         category_name=InfectionService.display_name
     ).order_by('created')

     patients_seen = set()
     serialise = []

     for episode in episodes:
         if episode.patient_id in patients_seen:
             continue
         serialise.append(episode)
         patients_seen.add(episode.patient_id)

    return serialise
```